AboutMe.Web/Program.cs

```diff
@@ -0,0 +1,49 @@
+using AboutMe.HelperExtensions;
+using Infrastructure;
+using Application.AppStart;
+using Core.Entities;
+using Microsoft.AspNetCore.Identity;
+using Core.Mapping;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages();
+builder.Services.AddContext(builder.Configuration.GetConnectionString("DefaultConnection"));
+builder.Services.AddServices();
+builder.Services.AddAutoMapper(typeof(AutoMapperProfile));
+
+builder.Services.AddIdentity<User, Role>(options =>
+{
+    options.Password.RequireDigit = false;
+    options.Password.RequireLowercase = false;
+    options.Password.RequireNonAlphanumeric = false;
+    options.Password.RequireUppercase = false;
+}).AddEntityFrameworkStores<AppDbContext>().AddDefaultTokenProviders();
+
+
+var app = builder.Build();
+
+app.SetupDataBase();
+
+if (!app.Environment.IsDevelopment())
+{
+    app.UseExceptionHandler("/Error");
+    app.UseHsts();
+}
+
+//app.UseHttpsRedirection();
+app.UseStaticFiles();
```

Comment:
also split this file into sections with functions so we can visually see what is happening here